heron/ui/src/python/BUILD

```diff
@@ -13,6 +13,9 @@ pex_library(
         "requests==2.3.0",
         "tornado==4.0.2",
     ],
+    deps = [
+        "//heron/common/src/python:common",
```

Comment:
You should create a new pex_library rule for these code moved from heron-ui and depend on that new rule only.